### ISSUE TITLE
Client File Access

### ISSUE THREAD

**Issue by danielpeintner** [jpro-one/jpro-issues#2]
I assume there is a way to access files on the client side and sending it to the server (which then processes the file accordingly). The javdoc for [FileUploader](https://www.jpro.one/api/2018.1.0/com/jpro/webapi/WebAPI.FileUploader.html) indicates that also.

What I miss is the documentation how to use it. I also checked the examples without success.
Is this something you want to make publicly available?

**Comment by FlorianKirmaier** [comment]
I have updated the [jpro webapi](https://www.jpro.one/?page=docs/current/api) of the [jpro.one](www.jpro.one) page, because it wasn't up to date. The Changes are relevant for the FileUploader.

How to use the FileUploader:
```
WebAPI webAPI = WebAPI.getWebAPI(stage);
WebAPI.FileUploader uploader = webAPI.makeFileUploadNode(this);
uploader.setSelectFileOnClick(true);
uploader.setOnFileSelected(new WebAPI.FileSelectedListener { s -> uploader.uploadFile(); }
uploader.uploadedFileProperty().addListener((prop,old,new) -> doSomethingWithFile(new));
```

I keep this sample open, until we publish a more detailed example.

**Comment by FlorianKirmaier** [comment]
@danielpeintner 
I added a new project, which contains various samples.
It contains a sample about file-access.

https://github.com/jpro-one/JPro-Samples

**Comment by danielpeintner** [comment]
Thanks!
Minor... clicking and selecting a file works like a charm. Dropping a file works on Chrome but not on Firefox61 (at least on my machine, Windows7)

**Comment by FlorianKirmaier** [comment]
@danielpeintner
Thank you for testing it.
I can confirm it's not working on Firefox. The file-drag-effect is also wrong.

**Comment by FlorianKirmaier** [comment]
@danielpeintner 
It's fixed in the latest release 2018.1.5 !
